### PR TITLE
Add sonarcloud

### PR DESCRIPTION
**Pull request by alphakennyn**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,13 +7,20 @@ cache:
   yarn: true
   directories:
   - node_modules
+addons:
+  sonarcloud:
+    organization: "dannylekim"
+    token:
+      secure: "FffhrC6wDSv2Mm5FxHtyryVYWWMGUj7aqPo+9/apGRsQcL/YKwD8EcIsS4oEGvh0UQCSmjZTdLoyO37DBFbeBaKEFKjqWxFkYw2+5d25YYqIBtStZimM4zg7tLRztk6fevEnvkqaoZkK2nkDtfDVFrhktscvhxZwWPdu4nKHS0eeGZaypZ9Gubp8js5aOXJIlrmS3bF6GpMgz7x4YGpwEU0usog5ianlGpr25ehdWjBd6JfVeX/vOy/efXdDiMDEZa2R4a8WpA4rJ1FO/bwz2oT6tY9EmvSuJykXTlrgZQ0FAYENcy+v9r0v0zHak7Vw/nHMhiqgG3NjgjYAATJkR/iJ5nsFo0uHo1p+2ua6CUetceUiDRSFdp4NEWpSWDum+56RR2Ebn3ID5pWjDsvt722nmjKucvqqJuXN37i4RkQJupN2PlGbsfaJ+SYyx+KabIE0DVcZ3bqUmhzK9igMxUekqj4I/rEnZrnMPkLiBEA+h+yk1JIUbA6ZxgiTwfOEFx5AIXMNvsiCBu974jFqOdtXfvOvkL7RyvQiue74qCuTtJ9Aw9gTHl5bdP1ZHne+fiuw6RfUuyEz4MS/6SmBFhYWLus6yrPmHJeuuRF/0HDBb3kqub4P68GkIzh9yqJtDoqUeT2DWS8yi39M717GMpI6fwIUpGkhM40LGemXq7E="
```
Comment (dannylekim) [inline .travis.yml:14]: There's no security concern having this token out like this right?

File: .travis.yml [hunk 2]
```diff
@@ -7,13 +7,20 @@ cache:
   yarn: true
   directories:
   - node_modules
+addons:
+  sonarcloud:
+    organization: "dannylekim"
```
Comment (dannylekim) [inline .travis.yml:12]: pretty sure the organization is Billsnap 

image

File: sonar-project.properties [hunk 3]
```diff
@@ -0,0 +1,12 @@
+sonar.projectKey=dannylekim_billsnapFront
+sonar.organization=dannylekim
```
Comment (dannylekim) [inline sonar-project.properties:2]: Billsnap

File: sonar-project.properties [hunk 4]
```diff
@@ -0,0 +1,12 @@
+sonar.projectKey=dannylekim_billsnapFront
```
Comment (dannylekim) [inline sonar-project.properties:1]: what dis, should it be Billsnap_billsnapFront

File: .travis.yml (hunk 2 above)
Comment (alphakennyn) [reply .travis.yml:12]: Odd. Sonarcloud gave me this to copy paste.

File: .travis.yml (hunk 1 above)
Comment (alphakennyn) [reply .travis.yml:14]: This is actually the encryted version of our key

File: .travis.yml (hunk 2 above)
Comment (dannylekim) [reply .travis.yml:12]: yeah it used to be dannylekim but I changed it, so I guess it should be billsnap now?

**Pull request closed without merging**